modules/virt-booting-vms-uefi-mode.adoc

```diff
@@ -16,7 +16,7 @@ You can configure a virtual machine to boot in EFI mode by editing the VM manife
 
 . Create a YAML file that defines a VM object. Use the firmware stanza of the example YAML file:
```

Comment:
Suggestion:

Edit or create a `VirtualMachine` manifest file. Use the `spec.firmware.bootloader` stanza to configure UEFI mode: